scripts/setup-vm.sh

```
@@ -55,9 +55,9 @@ fi
 # Codex and Claude Code use per-user credentials, so install their standalone
 # distributions as the account that will run Machinist.
 runuser -u "$runtime_user" -- env HOME="$runtime_home" \
-  bash -c 'curl -fsSL https://chatgpt.com/codex/install.sh | sh'
+  bash -c 'cd "$HOME"; curl -fsSL https://chatgpt.com/codex/install.sh | sh'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)